src/guide/admin/analysis-definition.md

```diff
@@ -0,0 +1,392 @@
+```yaml
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: <analysis deployment name>
+  namespace: default
+  labels:
+    app: <analysis deployment name>
+    component: flame-analysis
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: <analysis deployment name>
+      component: flame-analysis
+  template:
+    metadata:
+      labels:
+        app: <analysis deployment name>
+        component: flame-analysis
+    spec:
+      containers:
+        - name: <analysis deployment name>
+          image: <URL to image in Harbor>
+          ports:
+            - containerPort: 8000
+              protocol: TCP
+          env:
+            - name: DATA_SOURCE_TOKEN
+              value: none_needed
+            - name: KEYCLOAK_TOKEN
+              value: <randomly generated token>
+            - name: ANALYSIS_ID
+              value: <analysis UUID>
+            - name: PROJECT_ID
+              value: <project UUID>
+            - name: DEPLOYMENT_NAME
+              value: <analysis deployment name>
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          imagePullPolicy: IfNotPresent
+      restartPolicy: Always
+      terminationGracePeriodSeconds: 30
+      dnsPolicy: ClusterFirst
+      securityContext: {}
+      imagePullSecrets:
+        - name: flame-harbor-credentials
+      schedulerName: default-scheduler
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 25%
+      maxSurge: 25%
+  revisionHistoryLimit: 10
+  progressDeadlineSeconds: 600
+status:
+  observedGeneration: 1
+  replicas: 1
+  updatedReplicas: 1
+  readyReplicas: 1
+  availableReplicas: 1
+  conditions:
+    - type: Available
+      status: 'True'
+      reason: MinimumReplicasAvailable
+      message: Deployment has minimum availability.
+    - type: Progressing
+      status: 'True'
+      reason: NewReplicaSetAvailable
+      message: >-
+        ReplicaSet "<analysis deployment name>"
+        has successfully progressed.
+```
+
+### Nginx Deployment
+Nginx serves as a sidecar proxy, acting as the *only* means of communication between the analysis pod and everything else. The partnered analysis container can communicate only through the endpoints defined within this Nginx deployment (endpoint descriptions can be found in the [ConfigMap](#configmap) section), and like the analysis deployment, the same restrictive Network Policy is applied to the pods in this deployment through the use of specific labels and selectors.
+
+The created Nginx pod is only capable of communication with the partnered analysis pod, itself, and the DNS service for your kubernetes cluster. Because the analysis pod needs to communicate with other FLAME components, it must be able to discover those services within the kubernetes cluster and thus requires access to the kubernetes DNS service to resolve these service names.
```

Comment:
"The created Nginx pod is configured to only communicate with the partnered analysis pod, the other Flame components."

ther is no restriction from the network   policy on the nginx so it should not be hear